src/Servers/IIS/IntegrationTesting.IIS/src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj

```diff
@@ -35,15 +35,19 @@
           Condition="!Exists('$(AspNetCoreModuleV2ShimDll)') OR !Exists('$(AspNetCoreModuleV2OutOfProcessHandlerDll)')" />
   </Target>
 
+  <!-- Only build the target platform for product build (DotNetBuild=true). -->
```

Comment:
We can probably move this back into pass 1 in build.props now (i.e. undo some of my changes from https://github.com/dotnet/aspnetcore/pull/60896)